src/components/TodoComponents/Todo.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 
```

Comment:
`import  './Todo.css``

Reply:
This whole project looks good. Nice job getting the completed task toggle logic in there. 
If you import the CSS file with a short style parameter it works. 
I feel like you just didn't have time to do this. 
Nice.
```  
.completed {
    text-decoration: line-through;
    color: grey;
    opacity: .85;
}
```